src/simulator/base.py

```diff
@@ -0,0 +1,27 @@
+from object import Object, ObjectId
+from position import Position
+from shape import Shape
+from object_registry import ObjectRegistry
+from robot import Robot
+import numpy as np
+
+RadioID = str  # type aliasing, since we might change RadioID composition
+
+
+class Base(Object):
```

Comment:
One small thing, the documentation of the class is missing, although maybe you already had this in mind and left it for later.

Reply:
good point. I think it's bad documentation , isn't it? :|

Reply:
I mean in new commit.